prefs.ini

```diff
@@ -12,24 +12,14 @@ browser.safebrowsing.provider.mozilla.nextupdatetime = 1
 
 # For these prefs, the values need to be ADDED to the existing ones
 # instead of overwriting the existing values.
-#browser.safebrowsing.provider.mozilla.lists += moztestpub-track-digest256,moztestpub-trackwhite-digest256,mozstdstaging-track-digest256,mozfullstaging-track-digest256,mozstdstaging-trackwhite-digest256,base-track-digest256,baseeff-track-digest256,basew3c-track-digest256,content-track-digest256,contenteff-track-digest256,contentw3c-track-digest256
-browser.safebrowsing.provider.mozilla.lists = test-malware-simple,test-phish-simple,test-unwanted-simple,test-track-simple,test-trackwhite-simple,test-block-simple,goog-downloadwhite-digest256,mozstd-track-digest256,mozstd-trackwhite-digest256,mozfull-track-digest256,mozplugin-block-digest256,mozpub-track-digest256, moztestpub-track-digest256,moztestpub-trackwhite-digest256,mozstdstaging-track-digest256,mozfullstaging-track-digest256,mozstdstaging-trackwhite-digest256,base-track-digest256,baseeff-track-digest256,basew3c-track-digest256,content-track-digest256,contenteff-track-digest256,contentw3c-track-digest256,block-flash-digest256,except-flash-digest256,allow-flashallow-digest256,except-flashallow-digest256,block-flashsubdoc-digest256,except-flashinfobar-digest256, except-flashsubdoc-digest256, except-infobars-digest256
-#urlclassifier.disallow_completions += moztestpub-track-digest256,moztestpub-trackwhite-digest256,mozstdstaging-track-digest256,mozfullstaging-track-digest256,mozstdstaging-trackwhite-digest256,base-track-digest256,baseeff-track-digest256,basew3c-track-digest256,content-track-digest256,contenteff-track-digest256,contentw3c-track-digest256
-urlclassifier.disallow_completions = test-malware-simple,test-phish-simple,test-unwanted-simple,test-track-simple,test-trackwhite-simple,test-block-simple,goog-downloadwhite-digest256,mozstd-track-digest256,mozstd-trackwhite-digest256,mozfull-track-digest256,mozplugin-block-digest256,mozplugin2-block-digest256,moztestpub-track-digest256,moztestpub-trackwhite-digest256,mozstdstaging-track-digest256,mozfullstaging-track-digest256,mozstdstaging-trackwhite-digest256,base-track-digest256,baseeff-track-digest256,basew3c-track-digest256,content-track-digest256,contenteff-track-digest256,contentw3c-track-digest256,block-flash-digest256,except-flash-digest256,allow-flashallow-digest256,except-flashallow-digest256,block-flashsubdoc-digest256,except-flashsubdoc-digest256
+#browser.safebrowsing.provider.mozilla.lists += moztestpub-track-digest256,moztestpub-trackwhite-digest256,base-track-digest256,content-track-digest256
+
+#urlclassifier.disallow_completions += moztestpub-track-digest256,moztestpub-trackwhite-digest256,base-track-digest256,content-track-digest256
```

Comment:
Same - is this supposed to be commented out? Seems like it will have no effect now?

Reply:
I deleted everything except what was left on the meeting doc: https://docs.google.com/document/d/1jGBK-eKYBhlmJSeHu_nUyBz_SLcMSQn1YHoDPt2LJYU/edit

I WAS wondering if I should delete more selectively - but I wasn't sure myself.

Reply:
@ fmarier could you please weigh in? thanks

Reply:
Yes, the doc was probably not very clear.

These commented-out lines don't do anything by themselves. We do need to set the prefs explicitly. However, the prefs may change on the client, so it's worth referring back to the comment.

Here, what that means is: take the contents of `urlclassifier.disallow_completions` and add the following lists to it.